lib/mix/tasks/development_seeds.ex

```diff
@@ -49,8 +54,20 @@ defmodule Mix.Tasks.DevelopmentSeeds do
     ]
   end
 
+  def companies do
+    [
+      "thoughtbot",
+      "Vista",
+      "Microsoft",
+      "Citrix",
+      "WebAssign",
+      "BigLeap",
+      "EA",
+      "Big Corp",
+    ]
```

Comment:
Oh of course! That’s pretty important

> On Apr 22, 2016, at 1:20 PM, Jason Draper <notifications@github.com> wrote:
> 
> In lib/mix/tasks/development_seeds.ex <https://github.com/drapergeek/community/pull/21#discussion_r60773060>:
> 
> > @@ -49,8 +54,20 @@ defmodule Mix.Tasks.DevelopmentSeeds do
> >      ]
> >    end
> >  
> > +  def companies do
> > +    [
> > +      "thoughtbot",
> > +      "Vista",
> > +      "Microsoft",
> > +      "Citrix",
> > +      "WebAssign",
> > +      "BigLeap",
> > +      "EA",
> > +      "Big Corp",
> > +    ]
> It's minor but that wouldn't allow us to have spaces in the company names.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub <https://github.com/drapergeek/community/pull/21/files/cbbdbe24aa0ce14d2fb5275eb01c4a8e74596213#r60773060>